### DISCUSSION TITLE
HmIP-SLO und HmIP-eTRV-C-2

### DISCUSSION THREAD

**Discussion by rolfwer** [thkl/hap-homematic#561] [Q&A]
Mich würde interessieren, ob die beiden oben angegebenen Homematic IP Geräte mit dieser Erweiterung kompatibel sind.
Hat da schon jemand Erfahrung?
Vielen Dank vorab für die Info.

**Comment by Standarduser** [comment]
Der HmIP-SLO wird bei mir nicht angezeigt
